README.md

```diff
@@ -122,10 +99,10 @@ final_size(
   p_susceptibility,
   susceptibility
 )
-#>   demo_grp   susc_grp susceptibility  p_infected
-#> 1   [0,20) susc_grp_1            1.0 0.036325712
-#> 2  [20,40) susc_grp_1            0.5 0.009726187
-#> 3      40+ susc_grp_1            0.5 0.006240606
```

Comment:
There is an error in `data-raw/`, as you can see with the large change in the result here. You need to apply `t()` before scaling by the demography, not after.